pkg/console/install_panels.go

```diff
@@ -783,7 +814,7 @@ func addNetworkPanel(c *Console) error {
 		spinner := NewFocusSpinner(g, networkValidatorPanel, fmt.Sprintf("Requesting IP through DHCP..."))
 		spinner.Start()
 		go func(g *gocui.Gui) {
-			addr, err := getIPThroughDHCP(mgmtNetwork.Interface)
+			addr, err := getIPThroughDHCP(mgmtNetwork.Interfaces[0].Name)
```

Comment:
Why do we get IP for the slave interface?

Reply:
Because the bond network has not yet been created while selecting DHCP/Static method. I guess we can query for both master and slave interfaces and only report errors when both fail.

Reply:
We should get IP via the bonded device.
The MAC address that is used to request IP is determined by bond mode.

Reply:
I though it was only a DHCP test. If we really want to query DHCP server with bond network name, we should establish that beforehand, and that will reorder the whole network interface creation.

Reply:
What if we just remove this DHCP server validation, since the address is not stored anywhere. Is there any risk?

Reply:
We need to configure the network for next stages like fetching the remote config file. Can we create the bond after selecting dhcp/static mode?

Reply:
I moved DHCP check after the network `wicked ifreload` on 3b405e0. Thanks.